docs/mkcloud.md

```diff
@@ -71,8 +71,80 @@ To use mkcloud the following additional steps are needed:
 
 ## mkcloud
 
-`mkcloud` can be configured using different environment variables.  Some of
-them are listed below.  To get a complete list run: `./mkcloud -h`
+`mkcloud` consists of different steps which can be run independently, provided the order is logical.
+```
+   all -> expands to: cleanup prepare setupadmin addupdaterepo runupdate prepareinstcrowbar instcrowbar rebootcrowbar setupcompute instcompute proposal testsetup rebootcompute
```

Comment:
I'm not sure we want to duplicate all this expansion info, because it's too easy for it to get out of sync with the code. DRY!  Better to ensure that the user can find out what any step expands to by running `mkcloud` in a particular way.